src/content/docs/en/guides/rss.mdx

```diff
@@ -66,9 +67,11 @@ The package [`@astrojs/rss`](https://github.com/withastro/astro/tree/main/packag
     ```
 </Steps>
 
+<ReadMore>See the [`@astrojs/rss` README](https://github.com/withastro/astro/tree/main/packages/astro-rss) for the full configuration reference.</ReadMore>
+
 ## Generating `items`
 
-The `items` field accepts a list of RSS feed objects, which can be generated from content collections entries using `getCollection()` or from your page files using `pagesGlobToRssItems()`. 
+The `items` field accepts a list of RSS feed objects, which can be generated from content collections entries using `getCollection()` or from your page files using `pagesGlobToRssItems()`.
 
 The RSS feed standard format includes the following values for each published item:
```

Comment:
```suggestion
The RSS feed standard format includes metadata for each published item, including values such as:
```
So, my main concern is even with the link below, this sounds like an exhaustive, complete list. Anything to soften that up a bit here would be fine!